binding.gyp

```diff
@@ -5,7 +5,11 @@
       'includes': [ 'deps/snappy/common.gypi' ],
       'include_dirs': [ '<!(node -e "require(\'nan\')")', 'deps/snappy/<(os_include)' ],
       'dependencies': [ 'deps/snappy/snappy.gyp:snappy' ],
-      'sources': [ 'src/binding.cc' ]
+      'sources': [ 'src/binding.cc' ],
+      'xcode_settings': {
+        'WARNING_CFLAGS': [ '-Wno-sign-compare', '-Wno-unused-function' ],
+        'MACOSX_DEPLOYMENT_TARGET': '10.9'
+      }
```

Comment:
Note: without `MACOSX_DEPLOYMENT_TARGET` here, node-gyp fails with:

```
> snappy@6.3.1 install /Users/emadum/Code/forks/node-snappy
> prebuild-install || node-gyp rebuild

  CXX(target) Release/obj.target/snappy/deps/snappy/snappy-1.1.7/snappy-sinksource.o
  CXX(target) Release/obj.target/snappy/deps/snappy/snappy-1.1.7/snappy-stubs-internal.o
  CXX(target) Release/obj.target/snappy/deps/snappy/snappy-1.1.7/snappy.o
  LIBTOOL-STATIC Release/snappy.a
  CXX(target) Release/obj.target/binding/src/binding.o
clang: warning: include path for libstdc++ headers not found; pass '-stdlib=libc++' on the command line to use the libc++ standard library instead [-Wstdlibcxx-not-found]
In file included from ../src/binding.cc:1:
../src/./binding.h:5:10: fatal error: 'string' file not found
```

The `WARNING_CFLAGS` are not necessary for this, I copied them over to be consistent with the other .gyp file.